### ISSUE TITLE
Is it possible to output probability score for each token?

### ISSUE THREAD

**Issue by gccome** [dmis-lab/biobert#13]
Hello, most of the time it is important to also output the maximum probability score for each token prediction in NER. So I am just wondering can we output such score in run_ner.py?

**Comment by wonjininfo** [comment]
Hi, 
In our recent update(f8bdd2b), we added the feature which the model also outputs logits. 
You will find logits_test.txt file on <output_dir>. 
Thanks

**Comment by gccome** [comment]
Thanks for the update. Just have one quick question, what is the difference between `log_probs` and `probabilities`, why return 'log_probs' instead of `probabilities`?

**Comment by jhyuklee** [comment]
You can exponentiate log_probs to make them probs. It’s just more numerically stable when you work with very small probs.

**Comment by gccome** [comment]
Ah I see. thanks!

**Comment by luana-be** [comment]
Hi @jhyuklee, why isn't this modification on the master branch?
